index.js

```diff
@@ -33,32 +35,19 @@ app.put('/todos/complete', (req, res) => {
     const { id,completed } = req.body
```

Comment:
write statement if id exists in your data
 const result_of_id = data.findIndex(d=> d.id === todoId);
    console.log(result_of_id);

    if (result_of_id === -1){
        return  res.status(404).send({message: 'Todo not found'})
    }